projects/plugins/jetpack/modules/search/class-jetpack-search-customize.php

```diff
@@ -158,7 +158,7 @@ public function customize_register( $wp_customize ) {
 				$wp_customize,
 				$id,
 				array(
-					'description' => __( 'Choose post types to exclude from search results.', 'jetpack' ),
+					'description' => __( 'Choose post types to exclude from search results. For search to work properly, please leave at least one post type unchecked.', 'jetpack' ),
```

Comment:
I think we could be more direct here and say "you must...".

```suggestion
					'description' => __( 'Choose post types to exclude from search results. You must leave at least one post type unchecked.', 'jetpack' ),
```